src/pages/Login/Login.js

```diff
@@ -1,5 +1,48 @@
+import React from 'react';
+import './Login.css';
+import { useNavigate } from 'react-router-dom';
+import Header from '../../components/Header/Header';
+import Card from '../../components/Card/Card';
+import Button from '../../components/Button/Button';
+import Input from '../../components/Input/Input';
+
 function Login() {
+    const navigate = useNavigate();
+    return(
+        <>
+        <Header />
+            <div className='login-mainContainer'>
+                <div className='login-cardContainer'>
+                    <Card content={
+                        <div className='login-cardContent'>
+                            <div className = "login-textContainer">
+                                <h1 className='login-heading'>Taste<span>Buds</span></h1>
+                            </div>    
+                            <div className='login-username'>
+                                <Input className='login-username1' placeholder={'username'}
+                                />
+                            </div>
+                            <div className='login-password'>
+                                <Input placeholder={'password'}
+                                />   
+                            </div>
 
+                            <div className='login-buttonContainer'>
+                                <Button text={
+                                    <p className='buttonText'>Login</p>
```

Comment:
Replace this 'buttonText' classname with the updated name in Login.css file! (same for the className on line 36)